app/javascript/packs/card.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function(){
```

Comment:
make all functions arrow functions and rename this module to understand what does it stands for